src/components/AddEmailForm/AddEmailForm.jsx

```diff
@@ -31,38 +31,44 @@ const _AddEmailForm = ({ emailError, isAdding, wasSaved, add }) => {
 
   const formErrors = emailError ? [emailError] : errors;
 
-  return wasSaved ? (
-    <div className={`${styles.container} ${styles.NoMargin}`}>
-      <div className={styles.success}>
-        <h3 className={styles.title}>¡Todo listo!</h3>
-        <p className={styles.text}>Pronto te estaremos escribiendo.</p>
-      </div>
-    </div>
-  ) : (
-    <div className={styles.container}>
-      <p className={styles.title}>Dejanos tu e-mail y enterate al instante de las novedades!</p>
-      <form className={styles.form} onSubmit={onSubmit}>
-        <Input
-          placeholder="Tu e-mail"
-          value={email}
-          onChange={e => setEmail(e.target.value)}
-          hasError={!!formErrors}
-          disabled={isAdding}
-          autoFocus
-        />
-        {!!formErrors
-          ? formErrors.map(error => <ValidationError key={error}>{error}</ValidationError>)
-          : null}
-        <Button
-          type="submit"
-          color="secondary"
-          isLoading={isAdding}
-          disabled={isAdding}
-          className={styles.button}
-        >
-          Registrarme
-        </Button>
-      </form>
+  return (
+    <div className={`${styles.container} ${className ? className : ''}`}>
+      {wasSaved ? (
+        <div className={styles.success}>
+          <h3 className={styles.title}>¡Todo listo!</h3>
+          <p className={styles.text}>Pronto te estaremos escribiendo.</p>
+        </div>
+      ) : (
+        <Fragment>
```

Comment:
Isn't this the same as `<> ... </>`?

Reply:
Yup, you used to need a babel plugin for this so I kept using the long version. I'll change it.